test/fixtures/xml/worhseetWithSharedFormulas.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
It is not possible to reliably convince excel to save your formulas as shared (even if they are repeated). So, I am testing at the xml level for this one.